frontend/src/components/result/ResultSummary.jsx

```diff
@@ -20,14 +22,12 @@ const ResultSummary = (props) => {
           <dt className="col-sm-3">path name</dt>
           <dd className="col-sm-9">{result.pathName}</dd>
 
-          <dt className="col-sm-3">epoch</dt>
-          <dd className="col-sm-9">{lastLogDict.epoch}</dd>
-
-          <dt className="col-sm-3">iteration</dt>
-          <dd className="col-sm-9">{lastLogDict.iteration}</dd>
-
-          <dt className="col-sm-3">elapsed time</dt>
-          <dd className="col-sm-9">{lastLogDict.elapsed_time}</dd>
+          {logKeys.map((key) => (
+            <React.Fragment key={`summary-${key}`}>
+              <dt className="col-sm-3">{key.split('_').join(' ')}</dt>
```

Comment:
It is strange to use `split` and `join` for string replacement.
You should use `key.replace(/_/g, ' ')`.